.changelog/4562.yml

```diff
@@ -0,0 +1,6 @@
+changes:
+- description: Updated the update-release-notes command to handle changes in scripts and integration (using the objects under content_graph).
+                Updated the RN103 validation to validate the new fill in templates- "%%UPDATE_CONTENT_ITEM_CHANGE_DESCRIPTION%%",
+                        "%%UPDATE_CONTENT_ITEM_DESCRIPTION%%", "%%UPDATE_CONTENT_ITEM_NAME%%", "%%UPDATE_CONTENT_ITEM_TYPE%%"
```

Comment:
I recommend dividing this into two separate descriptions (see [here](https://confluence-dc.paloaltonetworks.com/display/DemistoContent/Changelogs) for more info):
```md
changes:
- description: Updated the ***update-release-notes*** command to handle changes in scripts and integration (using the objects under `content_graph`).
  type: feature
- description: Updated the *RN103* validation to validate the new fill in templates- "%%UPDATE_CONTENT_ITEM_CHANGE_DESCRIPTION%%", "%%UPDATE_CONTENT_ITEM_DESCRIPTION%%", "%%UPDATE_CONTENT_ITEM_NAME%%", "%%UPDATE_CONTENT_ITEM_TYPE%%".
  type: feature
pr_number: 4562
```